tests/ui_tests/test_cluster_list_page.py

```diff
@@ -1238,37 +1247,36 @@ def check_expectations():
                 for config_item in cluster_config_page.config.get_all_config_rows():
                     group_name = cluster_config_page.config.get_group_names()[0].text
                     assert group_name == 'group', "Should be group 'group' visible"
-                    if activatable:
-                        if not cluster_config_page.config.advanced:
-                            cluster_config_page.config.check_group_is_active(group_name, config['config'][0]['active'])
-                        if not field_invisible and (
-                            (cluster_config_page.config.advanced and field_advanced) or not field_advanced
-                        ):
-                            cluster_config_page.config.expand_or_close_group(group_name, expand=True)
-                            assert len(cluster_config_page.config.get_all_config_rows()) >= 2, "Field should be visible"
-                            if is_default:
-                                self.check_default_field_values_in_configs(
-                                    cluster_config_page, config_item, field_type, config
-                                )
-                            if is_read_only and config_item.tag_name == 'app-field':
-                                assert cluster_config_page.config.is_element_read_only(
-                                    config_item
-                                ), f"Config field {field_type} should be read only"
-                            if expected['alerts'] and not is_read_only:
-                                if field_type == "map":
-                                    is_advanced = cluster_config_page.config.advanced
-                                    cluster_config_page.driver.refresh()
-                                    if is_advanced:
-                                        cluster_config_page.config.click_on_advanced()
-                                    cluster_config_page.config.expand_or_close_group(group_name, expand=True)
-                                else:
-                                    cluster_config_page.config.click_on_advanced()
+                    if not activatable:
```

Comment:
but in this case we wont check 
assert len(cluster_config_page.config.get_all_config_rows()) == 1, "Field should not be visible"

Reply:
Why? This full block was inside of `if not activatable` condition. The line you're refering to won't be checked in previous version if activatable if False either